.travis.yml

```
@@ -1,10 +1,10 @@
 language: go
 
 go:
-  - 1.13
+  - 1.13.x
 
 env:
-  - GO111MODULE=on
+  - GO111MODULE=on SKIP_PREPARE_RELEASE_BINTRAY=true
```

Travis docs: [Global Variables](https://docs.travis-ci.com/user/environment-variables#global-variables)

---

should this be one variable per line?
```suggestion
  - GO111MODULE=on
  - SKIP_PREPARE_RELEASE_BINTRAY=true
```

---

Apparently not, although I originally thought the same. When I put them on separate lines Travis does two separate builds, each setting one of the variables.

---

[This](https://app.travis-ci.com/github/sky-uk/osprey/builds/238630586) was the previous build that kicked off builds for each of the `env` entries.

---

Ahh, I see :+1: